Gopkg.lock

```diff
@@ -838,35 +838,64 @@
     "informers/storage/v1alpha1",
     "informers/storage/v1beta1",
     "kubernetes",
+    "kubernetes/fake",
```

Comment:
I see changes to vendor metadata in the Gopkg.lock but no changes to the vendor directory. Is one of these not intentional? 

PS if you check in vendor changes, please do it in a different commit than your actual code for reviewer sanity.